<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QWidget" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Park &amp; Spark - Parking System</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="text">
    <string>Sign Up</string>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstNameInput">
   <property name="placeholderText">
    <string>First Name</string>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>760</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameInput">
   <property name="placeholderText">
    <string>Last Name</string>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>760</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailInput">
   <property name="placeholderText">
    <string>Email Address</string>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>760</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordInput">
   <property name="placeholderText">
    <string>Password</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>760</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="carTypeInput">
   <property name="placeholderText">
    <string>Car Type</string>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>760</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="carModelInput">
   <property name="placeholderText">
    <string>Car Model</string>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>760</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="text">
    <string>Register</string>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>760</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>760</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
